whats_fresh/whats_fresh_api/templates/base.html

```diff
@@ -9,8 +9,9 @@
         <link href='http://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,300italic' rel='stylesheet' type='text/css'>
         <link href='http://fonts.googleapis.com/css?family=Open+Sans:300&effect=3d-float' rel='stylesheet' type='text/css'>
         <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
-        <!-- // <script src="http://rawgit.com/Logicify/jquery-locationpicker-plugin/master/dist/locationpicker.jquery.js"></script> -->
-        <script src="http://rawgit.com/tschuy/jquery-locationpicker-plugin/master/src/locationpicker.jquery.js"></script>
+        <script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script>
+        <link href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/smoothness/jquery-ui.css" rel="stylesheet" type="text/css" media="all" />
+        <script src="//rawgit.com/tschuy/jquery-locationpicker-plugin/master/src/locationpicker.jquery.js"></script>
```

Comment:
Can you change all of the rest of the ``http://``s to ``//`` like you did with the rawgit link?